.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -25,7 +28,7 @@ https://github.com/TypeStrong/ts-node-repros
 
 ### Specifications
 
-* ts-node version: 
-* node version: 
-* Operating system and version: 
-* If Windows, are you using WSL or WSL2?: 
+* ts-node version:
+* node version:
+* Operating system and version:
+* If Windows, are you using WSL or WSL2?:
```

Comment:
My editor trimmed this due to `.editorconfig`; probably simpler to make this change than complicate the `.editorconfig`, but I'm happy to do either.